patches/chrome-android-java-src-org-chromium-chrome-browser-ntp-NewTabPageView.java.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPageView.java b/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPageView.java
+index d1ca3eb54eaa4988d2d347804152eae9d2381e00..c015ff58dd15669b3e6f26f8c95e57d5997dc3f8 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPageView.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/ntp/NewTabPageView.java
+@@ -99,7 +99,7 @@ public class NewTabPageView extends HistoryNavigationLayout {
+ 
+         // Don't attach now, the recyclerView itself will determine when to do it.
+         mNewTabPageLayout = (NewTabPageLayout) LayoutInflater.from(getContext())
+-                                    .inflate(R.layout.new_tab_page_layout, mRecyclerView, false);
```

Comment:
we should just over `new_tab_page_layout` with the contents of `brave_new_tab_page_layout`. Create new_tab_page_layout.xml in brave/android/java/res/layout and it will replace the chromium version of the file

Reply:
@bridiver i did the exact same thing in previous commits but as it's more like a different UI for newtabpage. updated it this way.

Reply:
I don't understand, we are completely replacing new_tab_page_layout, right? In that case we should use the same name file so we don't have to patch here

Reply:
@bridiver it created a patch like https://github.com/brave/brave-core/blob/e376bb777e6339e4e8210d95db313ec3f159ec1b/patches/chrome-android-java-res-layout-new_tab_page_layout.xml.patch so instead of adding patch like above i thought of creating a separate layout and just update it in java class.